src/pwa/services/webAuth.ts

```diff
@@ -22,42 +21,44 @@ export const verifyWebAuth = async (): Promise<boolean> => {
 
         return verifyAuthenticator(assertion);
     }
-    catch {
+    catch (error) {
+        console.info(error);
         return false;
     }
 }
 
 export const registerWebAuth = async (): Promise<boolean> => {
     const publicKeyCredentialCreationOptions: PublicKeyCredentialCreationOptions = {
-        challenge: base64ToBuffer(getChallenge()),
+        challenge: stringToBuffer(codeChallenge),
         rp: {
-            name: "IPS-AG WebAuth"
+            name: "IPS-AG WebAuth",
         },
         user: {
-            id: base64ToBuffer("ips-ag-user"),
+            id: stringToBuffer("ips-ag-user"),
             name: "pwa-user",
             displayName: "IPS-PWA",
         },
-        pubKeyCredParams: [{ alg: -7, type: "public-key" }, { alg: -257, type: "public-key" }],
+        pubKeyCredParams: [{ alg: -7, type: "public-key" }],
         authenticatorSelection: {
             authenticatorAttachment: "platform",
         },
-
-        timeout: 60000,
-        attestation: "none"
+        attestation: "direct"
     };
 
     try {
-        const credential = await navigator.credentials.create({
+        return await navigator.credentials.create({
             publicKey: publicKeyCredentialCreationOptions
-        });
-        registerAuthenticator(credential);
+        }).then(credential => {
+            console.log(credential);
```

Comment:
Guess this console log is used only for debugging purposes. It should be removed if so.

Reply:
thanks. I removed it.